cms/djangoapps/contentstore/views/tests/test_library.py

```diff
@@ -93,7 +119,7 @@ def test_lib_create_permission(self):
         self.client.logout()
         ns_user, password = self.create_non_staff_user()
         self.client.login(username=ns_user.username, password=password)
-
+        add_users(self.user, CourseCreatorRole(), ns_user)
```

Comment:
This has changed the test, so you need to update the docstring accordingly. ("Users who are given course creator roles should be able to create libraries.")

Please also add some new tests here that (like this test) actually use `self.client` to call the API. Your current tests don't actually test the API. You should copy this test and create versions which check:
* "Users who are not given course creator roles should still be able to create libraries if COURSE_CREATOR_GROUP is not enabled."
* "Users who are not given course creator roles should not be able to create libraries if COURSE_CREATOR_GROUP is enabled."
(You can use `ddt.data(True, False)` to combine those into one test)